src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--sun calendar--month-length--29">
+      <div class="calendar__day day"></div>
```

Comment:
one class is enough in this case

```suggestion
      <div class="calendar__day"></div>
```